calendar_reboot.js

```diff
@@ -1,304 +1,322 @@
-    };
-    
-    var removeFromCalendarDropdown = function(uniqueId) {
-        //remove the calendar with uniqueId from the saved calendars dropdown
-        //in the navbar
+        if (activeCalendarState !== null) {
+            var state = activeCalendarState;
+            var calendar = new Calendar(state, this);
+            console.log(calendar);
+            this.displayCalendar(calendar);
+        }
         
-        $('#calendarDropdown').children('#' + uniqueId).remove();
-    };
+    }
+    else {
+        this.buildCalendarForm.collapse('show');
+    }
     
     
-    var addFormError = function(id, srId) {
-        //adds the classes has-error and has-feedback
-        //on the div with the class form-group and it removes the hidden
-        //class on the span with class sr-only (for screen readers to notify
-        //that there has been an error
+}
+
+CheckIt.prototype.clearForm = function() {
+    // Clears the buildCalendar form.
+    removeFormErrors();
+    this.fullForm[0].reset();
+};
+
+CheckIt.prototype.createCalendar = function() {
+    // Create a new calendar and display it.
+
+    var title = this.calendarTitle.val();
+    var start = this.startDate.val();
+    var end = this.endDate.val();
+    
+    if (this.validateForm(start, end)) {
         
-        //id is the id of the eleent with the error glyphicon
-        //srId is the id for the span tag with the instructions for 
-        //screen readers, span has class sr-only
+        //clear the previously displayed calendar <-- or should i reload the page???
+        this.clearPage();
+
+        //make a calendar State
+        var state = emptyCalendarState({startDate: start, endDate: end, calendarTitle: title});
         
-        $(id).addClass('has-error has-feedback');
-        $(srId).removeClass('hidden');
+        //make calendar object
+
+        var calendar = new Calendar(state, this);
         
-    };
+        this.store.setActiveById(calendar.state.uniqueId);
+        this.store.save(calendar);
+
+        //add calendar to dropdown
+        this.addCalendarToDropdown(calendar.state.uniqueId, calendar.state.title);
     
-    var removeFormError = function(id, srId) {
-        //removes the classes has-error and has-feedback from the div
-        //with the class form-group, also adds the hidden class on the span
-        //with class sr-only (for screen readers)
-        
-        $(id).removeClass('has-error has-feedback');
-        $(srId).removeClass('hidden');
-    };
+        //build the calendar
+        this.buildCalendar(calendar);
+        this.buildCalendarForm.collapse('hide'); 
+   }
+};
+
+CheckIt.prototype.loadFromDropdown = function( event ) {
+    //when a calendar is clicked on in the dropdown menu
+    //the calendar that is currently on display (if there is one)
+    //needs to be removed and replaced with the calendar that was clicked on
+    //from the dropdown menu. Its' data will be loaded from localstorage
```

Comment:
No, you got rid of the sentence that says what this function does and kept the redundant and soon to be wrong one that says "data will be loaded from local storage where it was stored"